images/centos6.6/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
General comment about these Dockerfiles; can we either document that the "images" referred to in the file name refers only to the images of the KDC (i.e. not the node images) or maybe just change the name of the folder to kdc_images to be more explicit?

Reply:
renamed to kdc_images